src/test/resources/net/sf/jabref/importer/fileformat/RepecNepImporterTest3.bib

```diff
@@ -0,0 +1,6 @@
+@TechReport{,
+  month = {7},
```

Comment:
you're right. I don't know why it is imported wrong. In the other test the month was also wrong. In both it was one to low. So I've added  +1 for the month.
Now it should be correct. 
I've also added a field for the day.